scripts/mkcloud

```diff
@@ -466,7 +466,7 @@ function setupadmin
 
 function wait_for_node_shutdown
 {
-    wait_for 150 1 "LC_ALL=C virsh domstate $1 | grep shut.off" "$1 node to shut down"
+    wait_for 150 1 "LC_ALL=C virsh domstate $1 | grep 'shut off'" "$1 node to shut down"
```

Comment:
This only makes the test more restrictive, but doesn't match something that wasn't before. Example:
```
$ echo 'shut off'|grep shut.off && echo true
shut off
true
```